src/main/python/snakepit/main.py

```diff
@@ -80,6 +88,14 @@ def get_pypi_metadata(package, url='https://pypi.python.org/pypi/'):
                         format(url=url, package=package)).json()
 
 
+def custom_output_filename(filename, output_directory):
+    output_filename = "{0}/{1}".format(output_directory, filename)
```

Comment:
Please use `os.path.join` rather than `/`. `os.path` is imported as `osp` in this module.